pki/constants.go

```diff
@@ -21,6 +21,7 @@ const (
 	KubeSchedulerCertName      = "kube-scheduler"
 	KubeProxyCertName          = "kube-proxy"
 	KubeNodeCertName           = "kube-node"
+	KubeletCertName            = "kube-kubelet"
```

Comment:
the kube-node is supposed to be the kubelet cert, you should replace it if we are going to go with the cert per node solution since it won't be needed anymore

Reply:
I don't think we should merge the certificates because the kube-node certificate (and kubeconfig) is used by the kubelet to connect to the kube-api server. The kubelet certificate is what used to serve port 10250 on the kubelet and kube-api connects to. Besides the different function (auth vs server cert), it has different requirements which should not be merged into a single certificate.